ghost/admin/app/templates/application.hbs

```diff
@@ -14,6 +14,9 @@
             {{#if this.showBilling}}
                 <GhBillingModal @billingWindowOpen={{this.billing.billingWindowOpen}} />
             {{/if}}
+            {{#if (feature "exploreApp")}}
+                <GhExploreModal @exploreWindowOpen={{this.explore.exploreWindowOpen}} />
```

Comment:
Minor thing but there's no need to pass `@exploreWindowOpen` through as an argument, the component's `visibilityClass` getter can use `this.explore.exploreWindowOpen` directly and then there's no need to inject the service on the application controller.